python/sglang/srt/models/deepseek_v2.py

```diff
@@ -848,8 +848,8 @@ def forward_absorb(
                 self.w_vc.to(torch.bfloat16) * self.w_scale,
             )
         elif self.w_vc.dtype == torch.float8_e4m3fn:
-            attn_output_val, attn_output_scale = input_to_float8(
-                attn_output.transpose(0, 1), torch.float8_e4m3fn
+            attn_output_val, attn_output_scale = per_tensor_quant_fp8(
```

Comment:
Great work, btw can you fuse `transpose` and `contiguous` into Triton kernel too?

Reply:
> Great work, can you fuse `transpose` and `contiguous` into Triton kernel too?

Fusing `transpose` and `contiguous` with 3-d tensor is not a common case and I am going to impl another specialized kernel for it

Reply:
That's ok.